pkg/controller/kubedirectorcluster/members.go

```diff
@@ -240,6 +240,43 @@ func handleReadyMembers(
 	configmetaGenerator func(string) string,
 ) {
 
+	var connectionsVersion int64
+	var connVersionError error
+
+	if connectionsVersionStr, ok := cr.Annotations[shared.ConnectionsIncrementor]; ok {
+		connectionsVersion, connVersionError = strconv.ParseInt(connectionsVersionStr, 10, 64)
+
+		if connVersionError != nil {
+			shared.LogErrorf(
+				reqLogger,
+				connVersionError,
+				cr,
+				shared.EventReasonMember,
+				"Invalid connectionsIncrementor for role{%s}",
+				role.roleStatus.Name,
+			)
+			return
+
+		}
+
+		shared.LogInfo(
+			reqLogger,
+			cr,
+			shared.EventReasonCluster,
+			fmt.Sprintf("In HandleReadyMembers ConnectionsIncrementor value is: %d", connectionsVersion),
```

Comment:
we should only log when the connection changes else reconciler will flood the logs